geps/gep-1897/index.md

```diff
@@ -83,16 +87,16 @@ Gateway API is missing a mechanism for separately providing the details for the
 including (but not limited to):
 
 * intent to use TLS on the backend hop
-* client certificate of the gateway
-* system certificates to use in the absence of client certificates
+* CA certificates to trust
+* other properties of the TLS handshake, such as SNI and SAN validation
+* client certificate of the gateway (outside of scope for this GEP)
```

Comment:
System certificates should remain - this was added by request and is not going away.
Why is client certificate of the Gateway now outside of scope for this GEP?

Reply:
Because its not specified in the GEP...? Client certificate (mutual TLS) is covered in https://gateway-api.sigs.k8s.io/geps/gep-3155, not this GEP -- that doesn't change with this PR.

Maybe I misunderstood the question